src/libs/antares/study/include/antares/study/parts/common/cluster.h

```diff
@@ -135,9 +134,10 @@ class Cluster
     Matrix<> modulation;
 
 protected:
-    Data::ClusterName pName;
-    Data::ClusterName pID;
-    Data::ClusterName pGroup;
+    std::string pName;
+    std::string pID;
+
+    std::string group_ = "OTHER";
```

Comment:
Please make this private, use the getter/setter when needed.

This should ensure that the group name is always in UPPER CASE.